fuzzers/forkserver_libafl_cc/src/main.rs

```diff
@@ -86,7 +86,7 @@ struct Opt {
 
 #[allow(clippy::similar_names)]
 pub fn main() {
-    const MAP_SIZE: usize = EDGES_MAP_SIZE; //65536;
+    const MAP_SIZE: usize = EDGES_MAP_SIZE_IN_USE; //65536;
```

Comment:
Do we not support variable map sizes in libafl_cc forkserver yet? 
AFL++ collision free coverage at least can do that

Reply:
we do support but it is up to the harness if it uses it.

Reply:
When I try to set it to the 64K, it fails with:
thread 'main' panicked at /home/user/LibAFL/libafl/src/executors/forkserver.rs:765:13:
assertion failed: self.map_size.is_none() || map_size as usize <= self.map_size.unwrap()

I'm not sure why is this logic there, so I did not want to mess with it

Reply:
i fixed it in another PR already.